sdk/communication/communication-chat/recordings/browsers/chatclient_chat_operations/recording_successfully_creates_a_thread.json

```diff
@@ -1,51 +1,187 @@
-   "responseHeaders": {
-    "content-type": "application/json; charset=utf-8",
-    "location": "https://endpoint/chat/threads/19%3AV4_g-qkfJTO9AvFFHM9aLwnp8GiIbV1GZYCpRUH5jzI1@thread.v2"
-   }
-  }
- ],
- "uniqueTestInfo": {
-  "uniqueName": {},
-  "newDate": {}
- },
- "hash": "14f8ceacacd0b77b89cd04ef5e0f0262"
-}
\ No newline at end of file
+  "Entries": [
+    {
+      "RequestUri": "https://endpoint/identities?api-version=2021-03-07",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Accept": "*/*",
+        "Accept-Encoding": "gzip, deflate, br",
+        "Accept-Language": "en-US",
+        "Authorization": "Sanitized",
+        "Connection": "keep-alive",
+        "Content-Length": "34",
+        "Content-Type": "application/json",
+        "Origin": "http://localhost:9876",
+        "Referer": "http://localhost:9876/",
+        "sec-ch-ua": "",
+        "sec-ch-ua-mobile": "?0",
+        "sec-ch-ua-platform": "",
+        "Sec-Fetch-Dest": "empty",
+        "Sec-Fetch-Mode": "cors",
+        "Sec-Fetch-Site": "same-site",
+        "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/93.0.4577.0 Safari/537.36",
+        "x-ms-client-request-id": "378cda7e-4448-4e5e-90a8-be175c5cf3cc",
+        "x-ms-content-sha256": "WTRvgEjjVd\u002BbvyKw3WgXgDkU81aV8FWq\u002B4/BE\u002Bhe0\u002BA=",
+        "x-ms-date": "Wed, 09 Feb 2022 03:45:58 GMT",
+        "x-ms-useragent": "azsdk-js-communication-identity/1.0.0 core-http/1.2.6 OS/Linuxx86_64"
+      },
+      "RequestBody": {
+        "createTokenWithScopes": [
+          "chat"
+        ]
+      },
+      "StatusCode": 201,
+      "ResponseHeaders": {
+        "Access-Control-Allow-Origin": "*",
+        "api-supported-versions": "2020-07-20-preview2, 2021-02-22-preview1, 2021-03-07, 2021-03-31-preview1, 2021-10-31-preview, 2021-11-01, 2022-06-01",
+        "Content-Length": "290",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Wed, 09 Feb 2022 03:45:57 GMT",
+        "MS-CV": "/HWOQdMuRkSOAgmHG1NEMA.0",
+        "Request-Context": "appId=",
+        "Strict-Transport-Security": "max-age=2592000",
+        "X-Azure-Ref": "09jgDYgAAAAAj0M5t6spMSqMRuJrJdE5rV1NURURHRTA4MDcAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+        "X-Cache": "CONFIG_NOCACHE",
+        "x-ms-client-request-id": "378cda7e-4448-4e5e-90a8-be175c5cf3cc",
+        "X-Processing-Time": "72ms"
+      },
+      "ResponseBody": {
+        "identity": {
+          "id": "8:acs:987bcf3f-e830-40de-8e15-a91e0ccccb49_0000000f-7c78-2611-f40f-343a0d009ad2"
+        },
+        "accessToken": {
+          "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2NDQzODE5NTcuODk1fQ==.adM-ddBZZlQ1WlN3pdPBOF5G4Wh9iZpxNP_fSvpF4cWs",
```

Comment:
It looks like this used to be sanitized but isn't anymore.

Reply:
No, that's the fakeToken which is generated and also looks like a real token. 🥲
![image](https://user-images.githubusercontent.com/10452642/153311580-89ee6e0a-46d2-4f89-820d-e649ea2e1d4d.png)

A random string such as "Sanitized" didn't help, otherwise would've preferred that.

Reply:
Looking at the pipeline, credscan doesn't complain about this - so I think this should be good.

Right, @jeremymeng?

Reply:
CredScan isn't enabled for PR yet. There's an open PR which should be merged soon.

Reply:
I see you manually triggered it.

Reply:
Ah, okay. I hadn't opened up the token. It's fine then.